### PR TITLE
Fix typo in WrongTestMethodName description.

### PR DESCRIPTION
**Pull request by tasomaniac**

### CODE REVIEW

**Comment by tasomaniac** [comment]
Haha 😊 Will do. Tbh, I edited this on my phone. So I would have to do it later

File: README.md [hunk 1]
```diff
@@ -63,4 +63,4 @@ compile 'com.vanniktech:lint-rules-rxjava2:0.9.0-SNAPSHOT
 
 Copyright (C) 2017 Vanniktech - Niklas Baudy
 
-Licensed under the Apache License, Version 2.0
```
Comment (tasomaniac) [inline README.md]: Github seemed to automatically deleted an extra space char at the end.

**Comment by tasomaniac** [comment]
Thanks. I actually searched for it quickly but couldn't find it. I was looking at the tests for this particular check. Didn't see the registry test.

**Comment by vanniktech** [comment]
Yeah no worries. Thanks for the typo fix!

**Pull request merged**